tests/tasks/lib/test_tasks.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+import os
+import importlib
+from collections import Counter
+import numpy as np
+
+from jiant.shared import model_resolution
+from jiant.tasks import create_task_from_config_path
+from jiant.utils.testing.tokenizer import SimpleSpaceTokenizer
+
+CLASSIFICATION_TASKS = ['mnli', 'sst']
+
+
+@pytest.mark.parametrize("task_name", CLASSIFICATION_TASKS)
+def test_featurization_of_double_input_classification_task(task_name: str):
+    fixture_test_examples = importlib.import_module(
+        "tests.tasks.lib.resources.fixtures.task_examples." + task_name + "." + task_name + "_examples")
```

Comment:
Looking for feedback on a better way to import fixture data on a parameterized test run

Reply:
I recommend making a dictionary, and a get-function with a helpful error (to guide newcomers writing tests).

Reply:
Would the dict need to be updated each time a test is added or am I misunderstanding?